### ISSUE TITLE
docker-compose pull fails !!!!!!NOT INACTIVE IGNORE WRONG LABEL!!!!!!!  ALSO NO SETUP ISSUE SINCE IT IS NOT WINDOWS AS THE IDIOT TAGGING CLAIMS!!!!!!!!!!!!

### ISSUE THREAD

**Issue by manuelkamp** [TandoorRecipes/recipes#2586]
### Tandoor Version

0

### Setup

Docker / Docker-Compose

### Reverse Proxy

Nginx Proxy Manager (NPM)

### Bug description

root@srv-rezepte:~# docker-compose pull && docker-compose up -d                 [+] Running 13/16
 ⠦ web_recipes Pulling                                                    97.6s
   ⠿ 7264a8db6415 Already exists                                           0.0s
   ⠿ 66e1d5e70e42 Pull complete                                           10.7s
   ⠿ 2e5efaec6783 Pull complete                                           24.1s
   ⠿ bc20fd737683 Pull complete                                           24.2s
   ⠿ 19f128ef8d19 Pull complete                                           29.4s
   ⠿ ce00c9b07179 Pull complete                                           35.6s
   ⠿ 19ff44173d01 Pull complete                                           35.6s
   ⠿ 4f4fb700ef54 Pull complete                                           38.9s
   ⠿ b03e52565989 Pull complete                                           43.3s
   ⠿ 5029ffb7567e Pull complete                                           81.0s
   ⠿ 283f4480ccaf Pull complete                                           90.0s
   ⠿ 89ba79cd0a25 Pull complete                                           90.5s
   ⠸ c9d2930f3bd3 Extracting          154B/154B                           95.3s
   ⠸ f3cc611e75b1 Download complete                                       95.3s
 ⠿ nginx_recipes Pulled                                                    1.5s
failed to register layer: unlinkat /opt/recipes/.git/logs: invalid argument

**Comment by manuelkamp** [comment]
still not working

**Comment by mountaintime26** [comment]
Also Experience same issue 1.5.4->1.5.5 attempt:
`root@tandoortesting:~/tandoor# docker compose down && docker compose pull && docker compose up -d
[+] Running 4/4
 ✔ Container tandoor-nginx_recipes-1  Removed                                                         5.4s 
 ✔ Container tandoor-web_recipes-1    Removed                                                         5.1s 
 ✔ Container tandoor-db_recipes-1     Removed                                                         0.2s 
 ✔ Network tandoor_default            Removed                                                         0.3s 
[+] Pulling 29/3
 ✔ db_recipes 7 layers [⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                                              14.9s 
 ✔ nginx_recipes 8 layers [⣿⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                                           5.7s 
 ⠙ web_recipes 13 layers [⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿]    154B/154B    Pulling                                     25.1s 
failed to register layer: unlinkat /opt/recipes/.git/hooks: invalid argument`


Docker Version
`Server: Docker Engine - Community
 Engine:
  Version:          24.0.5
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.6
  Git commit:       a61e2b4
  Built:            Fri Jul 21 20:35:24 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.21
  GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
 runc:
  Version:          1.1.7
  GitCommit:        v1.1.7-0-g860f061
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0`

Docker Compose Version
`Docker Compose version v2.20.2`

**Comment by smilerz** [comment]
are you still experiencing this issue?

**Comment by mountaintime26** [comment]
> are you still experiencing this issue?

Yes:

`⠿ 84cfb9be9705 Extracting      [==================================================>]     154B/154B     failed to register layer: unlinkat /opt/recipes/.git/objects/pack: invalid argument`

**Comment by smilerz** [comment]
There’s something going on in your setup. Is it windows?  
Try looking [here](https://stackoverflow.com/questions/4452641/git-error-unable-to-unlink-old-some-file-name-bad-file-descriptor)  to see if it points you on a direction.

**Comment by manuelkamp** [comment]
yes, still having the problem as per today - I am not able to upgrade my docker installation. I was able to upgrade it for the last year, but not since I posted the initial error.
No, I am not using Windows - LXC Ubuntu 20.04. There is nothing other running on the LXC than docker with tandoor. I do not touch the container at all (other than apt update && apt upgrade and the docker commands to update tandoor: docker-compose down && docker-compose pull && docker-compose up -d)
